src/NodaTime.Web/Markdown/3.0.x/localdate-patterns.md

```diff
@@ -208,3 +208,25 @@ the inclusion of a leap month, the month name/number correspondence changes in a
 Until this is fixed, it is strongly recommended that you only use month *numbers* in any textual
 representations of dates in the Hebrew calendar. Additionally, you may wish to consider how to
 best clarify whether that month number is in the scriptural or civil numbering system.
+
+**Parsing and dependent values**
+
+Text parsing consists of two phases.
+
+The first phase obtains values from the text independently.
```

Comment:
Maybe "parses values from the text independently, left-to-right"?

Reply:
Done.